docs/content/concepts/cryptography/nautilus/nautilus-seal.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Seal Integration with Nautilus
+description: Learn about using Seal for secure secret use within Nautilus enclaves.
+keywords: [ seal, seal secret management, seal secrets, nautilus, enclaves ]
+---
+
+The Nautilus and [Seal](https://seal-docs.wal.app) integration pattern allows Nautilus enclaves to securely fetch public keys and identity-based decryption keys from Seal key servers. These keys can be used to encrypt or decrypt sensitive data directly inside the enclaves, ensuring that data is protected end-to-end, both at rest and in use.
+
+This pattern is ideal for use cases that require confidential data handling, secure AI workflows, or fine-grained access control policies enforced cryptographically.
+
+## What this enables
+
+Use this pattern to:
+
+* Decrypt sensitive data inside an enclave, without exposing secrets to the host.
+* Encrypt new data with public keys from Seal key servers, ensuring only authorized enclaves can later decrypt it.
+* Build trusted compute flows across secure storage (e.g., [Walrus](https://docs.wal.app)), on-chain objects (e.g., Sui), or other data sources.
+
+The encrypted data can arrive at the enclave from any source, as long as it was encrypted under the corresponding Seal access policy.
+
+## Why it matters
+
+* End-to-end protection: Data is encrypted before reaching the enclave and decrypted only within a verified environment.
```

Comment:
not sure if this is "why it matters".. but you can say this is a good pattern because you get the guarantees that only the enclave can have access to your secret